src/components/LineChart/index.tsx

```diff
@@ -224,7 +225,6 @@ export const LineChart = withTooltip<LineGraphType, DateValueType>(
         {tooltipData && (
           <div>
             <TooltipWithBounds
-              key={Math.random()}
```

Comment:
Hmmm good point. This wasn't introduced with this feature but I'll have a look! Thx

Reply:
I fixed it! there was a comparison checking if the mouse was nearer to the point right of the mouse or left of the mouse. The comparison was using the wrong comparison operation (< instead of >).